app/src/main/java/com/mensinator/app/settings/SettingsViewModel.kt

```diff
@@ -36,6 +36,8 @@ class SettingsViewModel(
             openColorPickerForSetting = null,
 
             daysBeforeReminder = -1,
+            periodNotificationMessage = "Period_Notification_Message",
```

Comment:
Alternatively I'm pretty sure this can start as "" since refreshData() will populate it with the value from the DB anyway

Reply:
Yep, please initialize it with ""

Reply:
Changed